wrangler-service/src/main/java/co/cask/wrangler/service/connections/ConnectionService.java

```diff
@@ -256,16 +260,13 @@ public boolean apply(@Nullable Connection input) {
       JsonObject response = new JsonObject();
       response.addProperty("status", HttpURLConnection.HTTP_OK);
       response.addProperty("message", "Success");
+      Optional<Connection> defaultConnection = getDefaultConnectionIfExists(connections);
+      if (defaultConnection.isPresent()) {
+        response.add("defaultConnection", getConnectionJsonObject(defaultConnection.get()));
```

Comment:
we shouldn't be returning the full object, just the id. Returning the full object can cause confusion if the information is ever different between what is returned here and what is in the list. It is also redundant.

Let's also just have the key as 'default'. I don't think 'Connection' is needed, as this is the response for the list of all connections.